docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
+* Show "No anomalies found" message instead of empty swim lane {kibana-pull}141098[#141098]
+* Explain Log Rate Spikes: Group results API {kibana-pull}140683[#140683]
+* Match Data Visualizer/Field stats table content with the popover {kibana-pull}140667[#140667]
+* Explain Log Rate Spikes: Adds discover link to analysis table {kibana-pull}139877[#139877]
+* Adding ecs_compatibility setting for find structure calls {kibana-pull}139708[#139708]
+* Improves messaging when an anomaly detection forecast errors {kibana-pull}139345[#139345]
+* Anomaly Detection: adds maps link when source data contains geo fields {kibana-pull}139333[#139333]
+* Quickly create ML jobs from lens visualizations {kibana-pull}136421[#136421]
+
+Management::
+* Url drilldown `date` helper now allows rounding up relative dates {kibana-pull}137874[#137874]
+* In CSV reports, an error message now appears on the job when fewer CSV rows are generated than expected {kibana-pull}137800[#137800]
+
+Maps::
+* Adds support for adhoc data views {kibana-pull}140858[#140858]
+* Timeslider control {kibana-pull}139228[#139228]
+* Support Vector tile runtime geo_point fields {kibana-pull}139047[#139047]
+* Show data view name in UI {kibana-pull}138928[#138928]
+* Adds ability to disable tooltips for layer {kibana-pull}138275[#138275]
+* Cancel button when editing by value from dashboard {kibana-pull}137880[#137880]
+
+Security::
+Audit events for user_login can be missing when using PKI or Kerberos {kibana-pull}139492[#139492]
+
+Uptime::
+* Project monitors - support lightweight project monitors {kibana-pull}141066[#141066]
+* Adds Actions popover menu {kibana-pull}136992[#136992]
+
+[float]
+[[fixes-v8.5.0]]
+=== Bug fixes
+Alerting::
+* Render the grid only if we have alerts {kibana-pull}142481[#142481]
+* Alerts Table browser field - fix siem browser fields call {kibana-pull}141431[#141431]
+* Adds getActionsHealth method to return permanent encryption key existence {kibana-pull}140535[#140535]
+* Clarify rule notification values {kibana-pull}140457[#140457]
+* Actions are not able to configure a max number of attempts {kibana-pull}138845[#138845]
+* Elasticsearch Query Rule doesn't have 'dark mode' view for query {kibana-pull}138631[#138631]
+* Getting error about secrets not being saved when import a SO (Connector Saved Object) {kibana-pull}138019[#138019]
+* Provide indication of how many rules are using connector on Connector List view {kibana-pull}137181[#137181]
+
+APM::
+* Remove check for infra data {kibana-pull}142835[#142835]
+* Prefer span metrics over span events {kibana-pull}141519[#141519]
+* Fixes search bar suggestions {kibana-pull}141101[#141101]
+* Sort trace samples {kibana-pull}140589[#140589]
+
+Dashboard::
+Fixes pinned filters that backed up in Session Storage {kibana-pull}142262[#142262]
+
+Discover::
+* Fixes an issue where histogram does not fill its container when time interval is not auto {kibana-pull}143004[#143004]
```

Comment:
Can be removed because it fixes a bug that didn't escape into the wilderness
```suggestion
```

Reply:
it seems it was resolved but still `Fixes an issue where histogram does not fill its container when time interval is not auto {kibana-pull}143004[#143004]` is here, but it can be remove